trending_data/src/main/webapp/init_map.js

```diff
@@ -4,20 +4,28 @@ let countriesWithSomeData;
 
 import {UniqueWindowHandler} from './unique_window_handler.js';
 import {getSupportedCountries} from './get_supported_countries.js';
-import {standard, darkerStandard} from './map_styles.js';
+import {standard, darkerStandard, darkMode} from './map_styles.js';
 
 /* eslint-disable no-unused-vars */
 /**
  * Initialise map
  */
 export async function initMap() {
   const initPos = new google.maps.LatLng(0, 0);
+  const darkModeType = new google.maps.StyledMapType( darkMode,
+      {name: 'Dark Mode'});
   map = new google.maps.Map(document.getElementById('map'), {
     center: initPos,
     zoom: 3,
     minZoom: 2,
     styles: standard,
+    mapTypeControlOptions: {
+      mapTypeIds: ['roadmap', 'satellite', 'hybrid',
+        'dark_mode'],
+    },
   });
+  map.mapTypes.set('dark_mode', darkModeType);
```

Comment:
I think dark mode shouldn't be the default - the user should have a choice to enable it.